packages/charts/src/scales/scale_continuous.ts

```diff
@@ -290,7 +301,7 @@ function getLinearNonDenserTicks(
     currentCount--;
     ticks = getLinearTicks(start, stop, currentCount, base);
   }
-  return ticks;
+  return integersOnly ? [...ticks.filter((v) => v % 1 === 0)] : ticks;
```

Comment:
you don't the spread operator here
```suggestion
  return integersOnly ? ticks.filter((v) => v % 1 === 0) : ticks;
```

Reply:
or even reusing the same functionality used before:
```suggestion
  return integersOnly ? ticks.filter(Number.isInteger) : ticks;
```